include/otf2xx/definition/container.hpp

```diff
@@ -146,21 +138,16 @@ namespace definition
         {
             return data
                 .emplace(std::piecewise_construct, std::forward_as_tuple(ref),
-                         std::forward_as_tuple(ref, args...))
+                         std::forward_as_tuple(ref, std::forward<Args>(args)...))
                 .first->second;
         }
 
-        void add_definition(Definition&& def)
+        void add_definition(Definition def)
```

Comment:
Technically speaking, taking parameters by value instead of providing both overloads is a pessimization, because you end up with more moves as would be necessary. ...but I'm beyond arguing for that.

Reply:
That's why I explained that in the PR description with the question if the 1 saved move is worth the overload or not.

Reply:
Honestly, I don't care. I find the failing test more interesting because on my machine all tests run fine.

Reply:
My bad, I had a typo. Fixed that.